recipes-core/cvm-reverse-proxy/cvm-reverse-proxy_dev.bb

```diff
@@ -0,0 +1,19 @@
+inherit go-mod
+LICENSE = "CLOSED"
```

Comment:
it's AGPL right?
What are the available options for license?

Reply:
good catch. 
I just moved it from the meta-evm layer to here as is.
I can look into it but IIRC Frieder wrote this [reverse proxy](https://github.com/konvera/cvm-reverse-proxy) from scratch and only used the constellation dependencies without changing them.

But good point, I will add the license just in case